.cmake-format.yaml

```
@@ -0,0 +1,4 @@
+format:
+  line_width: 120
+  tab_size: 4
+  dangle_parens: true
```

This should help address some of @mpusz's concerns.
```suggestion
parse:
  additional_commands:
    enable_iwyu:
      kwargs:
        MAPPING_FILE: 1
        MAX_LINE_LENGTH: 1
format:
  line_width: 120
  tab_size: 4
  max_subgroups_hwrap: 3
  max_pargs_hwrap: 6
  max_rows_cmdline: 2
  dangle_parens: true
  min_prefix_chars: 8
  max_prefix_chars: 8
  max_lines_hwrap: 2
markup:
  enable_markup: false
```

---

Great, thx